attributes/response/cddl/2_delegation.cddl

```diff
@@ -0,0 +1 @@
+delegation@response-attribute-arg = delegation-certificate / [ + delegation-certificate ]
```

Comment:
I'll comment here only once, but we MIGHT want to put an upper limit on the certification chain.

I wonder what's the performance impact on the server of maxing a block size with a very long certificate chain.

That makes me think we should probably create a "performance" test suite with those edge cases.

Reply:
Yeah I'll leave this to servers. There is an upper side limit to the amount of certificates (message size) but it does take time to process each of them so there's some DoS possible.